models/DataObject/ClassDefinition/Data/ManyToManyObjectRelation.php

```diff
@@ -728,9 +728,16 @@ public function addListingFilter(DataObject\Listing $listing, float|array|int|st
      */
     public function getFilterConditionExt(mixed $value, string $operator, array $params = []): string
     {
+        $prefix = '';
         $name = $params['name'] ?: $this->name;
 
-        return $this->getRelationFilterCondition($value, $operator, $name);
+        if ($params['brickPrefix']) {
+            // The brick prefix is always quoted and with a dot suffix, so removing the first
+            // and second last character to unquote
```

Comment:
The exact pattern can be seen on https://github.com/pimcore/admin-ui-classic-bundle/blob/2a129aa07f2f20c365f88c205d60d244468ff711/src/Helper/GridHelperService.php#L302-L310